src/features/configTable/configTable.tsx

```diff
@@ -0,0 +1,313 @@
+              ", "
+            )}`
+          );
+        });
+      }
+    });
+
+    return errors;
+  };
+
+export default function EditableTableExample() {
+  const [
+    postConfig,
+    {
+      isLoading: postConfigIsLoading,
+      isSuccess: postConfigSuccess,
+      isError: postConfigIsError,
+      error: postConfigError,
+    },
+  ] = usePostConfigMutation();
+
+  const { data, isLoading: getConfigIsLoading } = useGetConfigQuery(null, {
+    refetchOnReconnect: true,
+    refetchOnMountOrArgChange: true,
+  });
+
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    if (postConfigSuccess) {
+      dispatch(
+        dispatchNotification({
+          id: "1",
+          title: "Config Submitted!",
+          message: "Configuration has been submitted successfully",
+          status: "success",
+          duration: NotificationDuration.Medium,
+          isClosable: true,
+          position: "bottom-right",
+        })
+      );
+    }
+  }, [postConfigSuccess, dispatch]);
+
+  const onSubmit = async (values: { table: Employee[] }) => {
+    await postConfig({ employees: values.table, throwError: false }).unwrap();
+  };
+  const onSubmitError = async (values: { table: Employee[] }) => {
+    await postConfig({ employees: values.table, throwError: true })
+      .unwrap()
+      .catch((e) => {
+        dispatch(dispatchNetworkError(e));
+      });
+  };
+
+  return (
+    <Form
+      onSubmit={onSubmit}
+      initialValues={{ table: data }}
+      mutators={{
+        ...arrayMutators,
+      }}
+      validate={primaryKeyValidator(["name", "surName"])}
+      render={({
+        handleSubmit,
+        form: {
+          mutators: { push, pop },
+        },
+        form,
+        submitting,
+        pristine,
+        hasValidationErrors,
+      }: FormRenderProps<{ table: Employee[] }>) => {
+        return (
+          <VStack as="form" onSubmit={handleSubmit} spacing={6}>
+            <Heading>Employee</Heading>
+            <HStack spacing={4}>
+              <Button
+                rightIcon={<FaPlus />}
+                onClick={() =>
+                  push("table", {
+                    name: "",
+                    surName: "",
+                    employed: false,
+                  })
+                }
+                colorScheme="green"
+                aria-label="Add Employee"
+              >
+                New Employee
+              </Button>
+              <Button
+                type="submit"
+                isDisabled={submitting || pristine || hasValidationErrors}
+                isLoading={submitting || postConfigIsLoading}
+              >
+                Submit
+              </Button>
+              <Button
+                isDisabled={submitting || pristine || hasValidationErrors}
+                isLoading={submitting || postConfigIsLoading}
+                onClick={() => onSubmitError(form.getState().values)}
```

Comment:
this is not ok as the Form doesn't know that you're submitting. you need to call into handleSubmit.

check FF documentation but this is not ok.

as an alternative, you can add a top level element (alongside the table), which is checked by the postHandler to fail.
like a checkbox.